test/tbb/test_limiter_node.cpp

```diff
@@ -605,18 +605,18 @@ TEST_CASE( "Deduction guides" ) {
 }
 #endif
 
+struct TestLargeStruct {
+    char bytes[512]{ 0 };
+};
+
 //! Test correct node deallocation while using small_object_pool.
 //! (see https://github.com/oneapi-src/oneTBB/issues/639)
 //! \brief \ref error_guessing
 TEST_CASE("Test correct node deallocation while using small_object_pool") {
-    struct TestLargeStruct {
-        char bytes[512]{ 0 };
-    };
-
     tbb::flow::graph graph;
-    tbb::flow::queue_node<TestLargeStruct> input_node{ graph };
-    tbb::flow::function_node<TestLargeStruct> func{ graph, tbb::flow::serial,
-        [](const TestLargeStruct& input) { return input; } };
```

Comment:
Why these changes are needed ?

Reply:
Not necessary. This is just code style change to make it look like the rest of the file.
For example, https://github.com/oneapi-src/oneTBB/blob/6ed3ba2f428cab180dd2518b155bb2fd7563e03f/test/tbb/test_limiter_node.cpp#L381